dev/build-nodejs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This script builds the Node.js Lambda layer for OpenTelemetry instrumentation
+#
+# The script expects the dependencies to be cloned in specific locations
+# relative to this repository's root directory.
+
+set -euo pipefail
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+# Expected by build_nodejs_layer.sh
+export OPENTELEMETRY_JS_CONTRIB_PATH="$ROOT_DIR/../opentelemetry-js-contrib-cx"
+export OPENTELEMETRY_JS_PATH="$ROOT_DIR/../oss/opentelemetry-js"
+export IITM_PATH="$ROOT_DIR/../import-in-the-middle"
```

Comment:
Could we check for the present of these env vars before potentially re-exporting them? For instance, I like to use `.envrc` files (see [direnv](https://direnv.net)) to store env vars and this script would drop my env vars in favor of the hardcoded ones here.